UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>This is a good example of something that could be explained in the introduction. So we want to make a color-color diagram of two different populations: 1) the TW Hydra Asssociation, young stars whose coordinates are provided in a text file, and 2) a population of nearby M-dwarfs, old stars, the coordinates of which we'll get from a published catalog in Vizier. For both populations, we'll get the FUV, NUV magnitudes from GUVCat and the K, V magnitudes from Simbad.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>